SU-HSDO/suhumsci
# SHS-4997: Link field should generate URL to file, not media entity

# READY FOR REVIEW

## Summary
https://app.clickup.com/t/36718269/SHS-4997

LinkIt profile uses directly link instead of `/media/{node}`
Files linked through Link field  are displayed in browser and can be downloaded

## Need Review By (Date)
_['10/30', 'asap', etc.]_

## Urgency
_['low', 'medium', 'high', etc.]_

## Steps to Test
1. Check that LinkIt Matcher `/admin/config/content/linkit/manage/default/matchers` uses `Direct Dowload URL for media item`
2. Content > Create Flexible Page
3. Add a Postcad component
4. In a postcard link field start typing "pdf", then select one of the options
5. Click link in published content
6. Redirects to file in browser and has the option to download

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
